src/pages/home/report/ReportActionsView.js

```diff
@@ -245,6 +248,50 @@ function ReportActionsView(props) {
         }
     }, [hasCachedActions]);
 
+    const reportActionsToDisplay = useMemo(() => {
```

Comment:
@dukenv0307 Can we add clear comments here?
Like `What are we doing here?` and `Why are we doing this?` and link the decision GH comments as required.
Since this is not a straightforward approach and would be helpful for people seeing this at future